docs/docsite/README.md

```diff
@@ -1,7 +1,7 @@
 Ansible documentation
 =====================
 
-This project hosts the source behind [docs.ansible.com](https://docs.ansible.com/).
+This project hosts the source behind the general pages of [docs.ansible.com](https://docs.ansible.com/). Module-specific documentation is hosted in the various collections repositories. See [Ansible Galaxy](https://galaxy.ansible.com/), the list of [Ansible-maintained collections](https://docs.ansible.com/ansible/devel/community/contributing_maintained_collections.html), and the [ansible-collections organization](https://github.com/ansible-collections) for collections sources.
```

Comment:
Maybe change the wording of Ansible-maintained to Ansible Community maintained... something like that?

@gundalow, what do you think?

Reply:
@thaumos the Ansible-maintained collections are specifically the collections maintained in-house - network, security, cloud, etc.  They have specific contributing guidelines.

Reply:
@thaumos see https://docs.ansible.com/ansible/devel/community/contributing_maintained_collections.html for details. This page specifically lists the collections we maintain, along with related community content.